modules/oe_editorial_corporate_workflow/oe_editorial_corporate_workflow.module

```diff
@@ -15,14 +15,28 @@ use Drupal\node\NodeInterface;
  */
 function oe_editorial_corporate_workflow_menu_local_tasks_alter(&$data): void {
   $node = \Drupal::routeMatch()->getParameter('node');
-  if ($node instanceof NodeInterface && !empty($data['tabs'][0]['entity.node.edit_form']['#link'])) {
+  if (!$node instanceof NodeInterface) {
+    return;
+  }
+
+  // Edit tab.
+  if (!empty($data['tabs'][0]['entity.node.edit_form']['#link'])) {
     if (_oe_editorial_corporate_workflow_is_last_revision_published($node)) {
       $data['tabs'][0]['entity.node.edit_form']['#link']['title'] = t('New draft');
     }
     else {
       $data['tabs'][0]['entity.node.edit_form']['#link']['title'] = t('Edit draft');
     }
   }
+
+  // View tab.
+  if (!empty($data['tabs'][0]['entity.node.canonical']['#link'])) {
+    /** @var \Drupal\content_moderation\ModerationInformationInterface $moderation_info */
+    $moderation_info = \Drupal::service('content_moderation.moderation_information');
+    if ($moderation_info->isModeratedEntity($node) && $moderation_info->isDefaultRevisionPublished($node)) {
+      $data['tabs'][0]['entity.node.canonical']['#link']['title'] = t('View published');
```

Comment:
From what I have understood business just want to replace "View" with "View published" on every content type that has moderation enabled, while here we make the distinction of having it only when the default revision is published.

Let's just always use "View published" regardless of that condition, something like:
```php
if ($moderation_info->isModeratedEntity($node)) {
      $data['tabs'][0]['entity.node.canonical']['#link']['title'] = t('View published');
}
```

Reply:
I don't agree. See description of the ticket: This has to happen only on nodes which have a published revision.

Reply:
That's not the title of the issue but the name of this PR, which is taken automatically from the first commit. The title of the issue is:

```
Title: Rename View tab to View published on content types with moderation
Description: This has to happen only on nodes which have a published revision.
```

I still think it's better to be consistent any always use "View published" when you view a published revision.